spec/index.bs

```diff
@@ -4495,6 +4502,10 @@ enum GPUTextureAspect {
                                 |this|.{{GPUTexture/format}},
                                 |descriptor|.{{GPUTextureViewDescriptor/aspect}}).
 
+                        - |descriptor|.{{GPUTextureViewDescriptor/usage}} must be a subset of |this|.{{GPUTexture/usage}}.
+                        - If |descriptor|.{{GPUTextureViewDescriptor/usage}} includes the {{GPUTextureUsage/STORAGE_BINDING}} bit:
+                            - |descriptor|.{{GPUTextureViewDescriptor/format}} must be listed in [[#plain-color-formats]] table
+                                with {{GPUTextureUsage/STORAGE_BINDING}} capability for the appropriate access mode.
```

Comment:
Need a similar check for RENDER_ATTACHMENT usage. May be able to factor out that validation from `validating GPUTextureDescriptor` so it can be reused?

Reply:
Added. Didn't feel like factoring it out of `validating GPUTextureDescriptor` actually simplified thing much because that still needs to do dimension validation and the resulting text would probably end up almost as long.